My_Math.java

```diff
@@ -0,0 +1,19 @@
+public class My_Math
+
+{
+    public static double ceil(double n)
```

Comment:
//Ceil Method

Reply:
hey bro, pls tell difference between Math.rint() and Math.round()